packages/esm-reports-app/src/components/scheduled-report-status.component.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import styles from './reports.scss';
+import { useTranslation } from 'react-i18next';
+
+interface ReportStatusProps {
+  hasSchedule: string;
+}
+
+const ScheduledReportStatus: React.FC<ReportStatusProps> = ({ hasSchedule }) => {
+  const { t } = useTranslation();
+
+  const renderStatusLabel = () => {
+    if (hasSchedule) {
+      return <span className={styles.scheduledStatusText}>{t('scheduled', 'Scheduled')}</span>;
+    } else {
+      return <span className={styles.notScheduledStatusText}>{t('notScheduled', 'Not Scheduled')}</span>;
+    }
+  };
+
+  return renderStatusLabel();
```

Comment:
I think this can be simplified using classnames to something like:

```suggestion
  return (
    <span
      className={classNames({
        [styles.scheduledStatusText]: hasSchedule,
        [styles.notScheduledStatusText]: !hasSchedule,
      })}
    >
      {hasSchedule ? t('scheduled', 'Scheduled') : t('notScheduled', 'Not Scheduled')}
    </span>
  );
```

Reply:
fixed